components/cam/src/physics/crm/crm_physics.F90

```diff
@@ -411,6 +406,14 @@ subroutine crm_physics_tend(ztodt, state, tend, ptend, pbuf, cam_in, cam_out, &
    real(crm_rknd) :: crm_rotation_std     ! scaling factor for rotation (std dev of rotation angle)
    real(crm_rknd) :: crm_rotation_offset  ! offset to specify preferred rotation direction 
 #endif
+   integer, save :: icall = 0
+
+   ! CRM types
+   type(crm_state_type)  :: crm_state
```

Comment:
@xyuan , it looks like you're getting rid of passing this to the subroutine as an argument and instead storing it as a local data object upon initialization. Is that what you're doing? And if so, what is the  reason for doing this?